.yarnrc.yml

```
@@ -1 +1,2 @@
 nodeLinker: pnp
+nodeLinker: node-modules
```

YAMLlint (1.38.0)

[error] duplication of key "nodeLinker" in mapping

(key-duplicates)